### PR TITLE
[GUI] Fix project's Ellipsoid option description (adding angle to distance and area)

### PR DESCRIPTION
**Pull request by agiudiceandrea**
## Description

Also angle measurement, besides the distance and area ones, is ellipsoidal. (Unless "None \ Planimetric" ellipsoid is set).

In Project -> Properties -> General -> Measurements:
from
```
Ellipsoid
(for distance and area calculations)
```
to
```
Ellipsoid
(distance, area, angle calculations)
```
Another possible approach would be to completely remove the `(for distance and area calculations)` text.
Not sure if this PR is worth to be backported.

EDIT: as suggested in the review, I've removed the text in parenthesis and added a tooltip to the "Ellipsoid" option's label with the text `For distance, area and angle calculations`.

### CODE REVIEW

File: src/ui/qgsprojectpropertiesbase.ui [hunk 1]
```diff
@@ -605,7 +605,7 @@
                    <widget class="QLabel" name="textLabel1_8">
                     <property name="text">
                      <string>Ellipsoid
-(for distance and area calculations)</string>
+(distance, area, angle calculations)</string>
```
Comment (3nids) [inline src/ui/qgsprojectpropertiesbase.ui:608]:
```suggestion
(distance, area and angle calculations)</string>
```
?

Comment (agiudiceandrea) [reply src/ui/qgsprojectpropertiesbase.ui:608]: I tried not to make the text any longer.
Original:
image
My PR
image
Suggested change:
image

Comment (3nids) [reply src/ui/qgsprojectpropertiesbase.ui:608]: Remove the text in parenthesis and let's go for a tooltip?

Comment (agiudiceandrea) [reply src/ui/qgsprojectpropertiesbase.ui:608]: What do you think?
image
(The tooltip is only on the label, not on the combobox)

Comment (3nids) [reply src/ui/qgsprojectpropertiesbase.ui:608]: perfect!

Comment (agiudiceandrea) [reply src/ui/qgsprojectpropertiesbase.ui]: PR updated.

**Review by 3nids — APPROVED** [review]

**Pull request merged**